scripts/analyze.py

```diff
@@ -23,20 +23,20 @@
 # under the License.
 
 # Simple helper script to create graphs based on multiple
-# test_execution.json files (it's a summary of the results of
-# a single test_execution which is
-# stored in ~/.benchmark/benchmarks/test_executions/TEST_EXECUTION_TS/).
+# test_run.json files (it's a summary of the results of
+# a single test_run which is
+# stored in ~/.benchmark/benchmarks/test_runs/test_run_TS/).
```

Comment:
This can be corrected: `stored in ~/.benchmark/benchmarks/test_runs/<test_run_id>/)`